.changeset/gentle-mice-create.md

```diff
@@ -0,0 +1,7 @@
+---
+"@cloudflare/vite-plugin": patch
+"@cloudflare/unenv-preset": patch
```

Comment:
Should this be a `minor` for `@cloudflare/unenv-preset`?

Reply:
Very good question.

I am on the fence here... there is indeed a new API `getCloudflareContext()` so technically it should be minor but not real new functionality. I was thinking about bumping the minor when the returned preset actually changes with the passed date/flags.

I guess we can do both: minor for this PR and minor for the PR changing the presets later.

Thoughts?

Reply:
> Should this be a `minor` for `@cloudflare/unenv-preset`?

I have updated the PR to set a minor. I think it's best this way.